pkgs/test/nixos-functions/default.nix

```diff
@@ -24,13 +24,15 @@ in lib.optionalAttrs stdenv.hostPlatform.isLinux (
       system.nixos = dummyVersioning;
       boot.loader.grub.enable = false;
       fileSystems."/".device = "/dev/null";
+      system.stateVersion = lib.trivial.release;
```

Comment:
This one was redundant because of the default in `test-instrumentation.nix`. Removing in https://github.com/NixOS/nixpkgs/pull/171647.